package.json

```diff
@@ -113,7 +113,7 @@
     "babel-plugin-replace-identifiers": "^0.1.1",
     "cssnano": "^4.1.10",
     "doctoc": "^1.4.0",
-    "husky": "^4.2.2",
```

Comment:
This is the devDep that is used internally, and not the one that is in the templates. The one in the templates is here: https://github.com/formium/tsdx/blob/462af2d002987f985695b98400e0344b8f2754b7/src/templates/basic.ts#L6